src/schema/basic_slots.yaml

```diff
@@ -57,6 +57,7 @@ slots:
     description: >-
       An optional string that species the type object. 
```

Comment:
There is a typo "species" -> "specifies".  
Not sure how I missed that up until now :(

Reply:
Can you update the [CHANGELOG](https://github.com/microbiomedata/nmdc-schema/blob/main/CHANGELOG.md)?